clients/cloud/c/example.config

```diff
@@ -0,0 +1,5 @@
+bootstrap.servers=<BROKER_ENDPOINT>
```

Comment:
Some users thought these examples were for ccloud only, and so the README verbiage changed recently to make it more generic to clarify to users that these examples can be used for any Kafka cluster, not just CCloud.

To that end, should we create separate files, e.g.

* `example-ccloud.config`
* `exampe-localhost.config`

If we do consider this, we'll need to take into account whether CCloud expects a specific file name.